spec/lib/sidekiq/form526_job_status_tracker/job_tracker_spec.rb

```diff
@@ -3,7 +3,12 @@
 require 'rails_helper'
 
 describe Sidekiq::Form526JobStatusTracker::JobTracker do
-  let(:dummy_class) { Class.new { include Sidekiq::Form526JobStatusTracker::JobTracker } }
+  let(:dummy_class) do
```

Comment:
More descriptive variable name:
```suggestion
  let(:worker_class) do
```